api/agent/call.go

```diff
@@ -311,6 +311,9 @@ func buildConfigWithPath(app *models.App, fn *models.Fn, path string) models.Con
 	}
 
 	conf["FN_FORMAT"] = fn.Format
+	if fn.Format == models.FormatHTTPStream { // TODO should be always soon...
+		conf["FN_LISTENER"] = "unix:/iofs/lsnr.sock" // XXX(reed): hardcoding this is ok right? it's a contract
```

Comment:
shouldn't this be `unix:///iofs/lsnr.sock` or just `/iofs/lsnr.sock`, this would really simplify FDK life by avoiding necessary URL parsing.

Both Go HTTP server, Python aiohttp server and other just need a file path, so, they need to rip off `unix:/` prefix. Since that's a contract we know that it's gonna be a Unix socket only.

Reply:
having protocol seems like the right thing to do in case we add tcp support. urls are easy to parse and there are parsers available in every major language...